server.rb

```diff
@@ -0,0 +1,44 @@
+require 'sinatra'
+require 'pry'
+require 'csv'
+
+configure do
+  enable :sessions
+end
+
+articles_array = []
+
+get '/articles' do
+  # binding.pry
+  articles_array = CSV.read("./articles.csv")
+  erb :news_home, locals: { articles_array: articles_array }
+end
+
+# get '/articles/new' do
+#   redirect '/articles/new/error_pass' #redundant
+# end
+
+get '/articles/new' do
+  erb :submissions, locals: {error_code: session["error"]}
+end
+
+
+post '/articles/new' do
+  if params["title"] == "" || params["URL"] == "" || params["description"] == ""
+    # $error_code = "empty_string"
+    session["error"] = "empty_string"
```

Comment:
it is always nice to give guidance of what was wrong

Opportunity to write a method that tells you which of the params was empty and update "empty_string" to "missing URL" or "missing title"